cmd/netobserv-ebpf-agent.go

```diff
@@ -39,22 +53,30 @@ func main() {
 
 	logrus.WithField("configuration", fmt.Sprintf("%#v", config)).Debugf("configuration loaded")
 
-	flowsAgent, err := agent.FlowsAgent(&config)
-	if err != nil {
-		logrus.WithError(err).Fatal("can't instantiate NetObserv eBPF Agent")
-	}
+	if config.EnablePCA {
+		if config.PCAFilters == "" {
+			logrus.Fatal("[PCA] NetObserv eBPF Agent instantiated without filters to identify packets. Please set environment variable PCA_FILTER.")
```

Comment:
Should we treat this for capture everything? That might be extremely resource intensive. This was a design decision we made.  This could be updated if the team deems this behavior correct.

Reply:
wanted to share one of the PM customers asked use case is to do capture all (just learnt this now) cc'd @jpinsonneau

Reply:
+1 it seems to be a use case now :upside_down_face: 
Would it work if we capture everything expect our own netobserv traffic ? Let's say with unlimited resources

Reply:
Updated to collect and export all packets if the filter (enviroment variable PCA_FILTER) is not set at all. 
However if the filter is wrongly set, incorrect format/ non-supported protocols etc, then the agent doesnt export anything.